Makefile

```diff
@@ -20,3 +20,7 @@ $(OUT)/segmented_cfmm.tz: ligo/**
 	mkdir -p $(OUT)
 	# ============== Compiling contract ============== #
 	$(BUILD) ligo/main.mligo main --output-file $(OUT)/segmented_cfmm.tz
+
```

Comment:
How about also adding a target to build the haskell library only?

It seem like it will soon be useful anyway.

Reply:
Added `lib` step to build just the library.